### ISSUE TITLE
td-json-open assertion exists multiple times

### ISSUE THREAD

**Issue by adriankast** [thingweb/thingweb-playground#124]
Due to the multiple usage of td-json-open and child assertions of it (td-json-open_xyz) in both automatic and manual assertions it is not merged correctly, which results in 337 assertions if a single TD is assertion tested. If multiple TDs are under test, the result merge probably also merges the duplicates and 335 assertions are reported.

**Comment by egekorkan** [comment]
@adriankast this is now fixed right?

**Comment by adriankast** [comment]
No, this was the issue we agreed to not fix at first but just write an issue, since you meant that this is kind of an structural problem an shouldn't just be "Bugfixed"

**Comment by FadySalama** [comment]
This assertion is causing problems for automatic generation of manual.csv! @egekorkan We should talk about it

**Comment by egekorkan** [comment]
I remember this discussion. It stems from the following part in the TD spec:

![image](https://user-images.githubusercontent.com/20195407/169694087-5af898fc-09ab-47e3-83a0-ef2938f55704.png)

This is annoying since the first one (parent assertion)

I will open an issue about it, not sure why I did not do it before already.

Regarding automatic generation of manual.csv, @FadySalama you can do a hacky solution for now by ignoring the children assertions, i.e. removing the children assertions in the input csv

**Comment by egekorkan** [comment]
also see https://github.com/w3c/wot-thing-description/issues/1511
